usagov/usagov-2021
# usagov-1323-focus-styles: Update public_analytics.css

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-

## Description

Previously this ticket removed the tab-index from the bar chart "Visits in the Past 30 Days." With that fixed we now need to update the css of the tab focus 

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



### Requires New Config

### Requires New Content

### Validation Steps
- You won't use localhost to test this but when you get the repo for the PR if using VSCode you can use the Live Server extension to open a live preview.
- In the preview you'll see the data isn't pulling from the database. For each "<ul class="data bar-chart">" you will see a data-source in the section element above it. Go to the link in the datasource and download that data to a json file locally. Replace all the links with references to your local files. (Tedious but best way to truly test this.) You should have 14 bar-charts with links. 
- Once the preview has rendered with the bar charts and urls tab through them. Make sure you can visually see a focus compared to only the right boxshadow we see in prod. 
- Before: 
![Screenshot 2024-01-29 at 2 45 31 PM](https://github.com/usagov/usagov-2021/assets/108532577/49d011d2-688e-4413-9021-2eb333c7b27d)

- After: 
![Screenshot 2024-01-29 at 2 45 13 PM](https://github.com/usagov/usagov-2021/assets/108532577/e5298d84-4ab0-402e-a630-48bcc660178f)


## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
